konig-spreadsheet/src/main/java/io/konig/spreadsheet/WorkbookLoader.java

```diff
@@ -2032,9 +2048,16 @@ private void loadClassRow(Row row) throws SpreadsheetException {
 					for (URI subclassId : subclassOf) {
 						graph.edge(classId, RDFS.SUBCLASSOF, subclassId);
 						graph.edge(subclassId, RDF.TYPE, OWL.CLASS);
+						if(termStatus != null) {
+							graph.edge(classId, XOWL.TERMSTATUS, termStatus);
```

Comment:
This is superfluous.  Why add this edge inside the loop of subclasses?

It is sufficient to add the edge once outside the loop, as you do below.